src/Services/MediaService.php

```diff
@@ -46,16 +51,27 @@ public function create($content, string $fileName, array $data = []): Model
     {
         $fileName = $this->saveFile($fileName, $content);
 
-        $preview = $this->createPreview($fileName);
+        if (!Arr::get($data, 'preview_drivers', false)) {
+            $previews = $this->createPreviews($fileName);
+        } else {
+            $previews = $this->createPreviews($fileName, $data['preview_drivers']);

```

Comment:
let's use one method call without wrap in if

```suggestion
            $previews = $this->createPreviews($fileName, Arr::get($data, 'preview_drivers'));
```